static/css/github-create-branch.css

```diff
@@ -90,4 +90,16 @@
 .error-message {
   color: #DE350B;
   font-size: 0.9rem;
+}
+
+.no-repo-container {
+  display: none;
+  margin: 4px 0 0;
+}
+.no-repo-link:hover {
```

Comment:
nit: spaces between rulesets for readability

Reply:
Spacing added